_posts/ArisWells/2021-12-06-ArisWells_Final_Project_Update

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Aris, could you add a `.md` extension to the file name so that all the good markdown you have written shows up?